plugins/CoreHome/vue/src/AjaxHelper/AjaxHelper.ts

```diff
@@ -358,13 +359,26 @@ export default class AjaxHelper<T = any> { // eslint-disable-line
     this.requestHandle = this.buildAjaxCall();
     window.globalAjaxQueue.push(this.requestHandle);
 
+    let $timeout: ITimeoutService|null = null;
+    try {
+      $timeout = Matomo.helper.getAngularDependency('$timeout');
+    } catch (e) {
+      // ignore
+    }
+
     const result: AbortablePromise<T> = new Promise<T>((resolve, reject) => {
-      this.requestHandle!.then(resolve).fail((xhr: jqXHR) => {
+      this.requestHandle!.then((...args) => {
+        resolve(...args);
```

Comment:
Seems the tests are failing because of this line
```
plugins/CoreHome/vue/src/AjaxHelper/AjaxHelper.ts:371:17 - error TS2556: A spread argument must either have a tuple type or be passed to a rest parameter.
371         resolve(...args);
```

Reply:
:+1: fixed in last commit